### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Finniki**
# Description

This PR adds the source files for this project

Fixes # (issue)

# Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Details

- ### **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Feature
- ### **What is the current behavior?** (You can also link to an open issue here)
The user can click keys to play the drums
## Screenshots (if applicable)

# Checklist:

- [ ] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: index-START.html [hunk 1]
```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
```
Comment (gertie-sheshe) [inline index-START.html]: I think you can rename this back to `index.html`?

File: index-START.html [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    <audio data-key="83" src="sounds/hihat.wav"></audio>
+    <audio data-key="68" src="sounds/kick.wav"></audio>
+    <audio data-key="70" src="sounds/openhat.wav"></audio>
+    <audio data-key="71" src="sounds/boom.wav"></audio>
+    <audio data-key="72" src="sounds/ride.wav"></audio>
+    <audio data-key="74" src="sounds/snare.wav"></audio>
+    <audio data-key="75" src="sounds/tom.wav"></audio>
+    <audio data-key="76" src="sounds/tink.wav"></audio>
+
+    <script>
```
Comment (gertie-sheshe) [inline index-START.html]: One last challenge 🙈 See if you can move this to an external `script.js` file and import it to this file

**Review by gertie-sheshe — APPROVED** [review]
Looks good and sounds good too! 😅 @Finniki Added two comments which are basically file and file structure changes. I hope you enjoyed and learned while working on this :)

File: index-START.html (hunk 1 above)
Comment (Finniki) [reply index-START.html]: done

File: index-START.html (hunk 2 above)
Comment (Finniki) [reply index-START.html]: Done

**Comment by Finniki** [comment]
Both comments addressed boss.

**Pull request merged**